### PR TITLE
[Add] Xiaomi icon

### PR DESCRIPTION
**Pull request by mudlabs**
**Issue:**
No related issue.

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- Manually vectorized
- Hex value taken from Xiaomi logo used in global website _(see data file)_.

### CODE REVIEW

File: icons/xiaomi.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (ericcornelissen) [inline icons/xiaomi.svg]:
```suggestion
<svg role="image" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment (mudlabs) [reply icons/xiaomi.svg]: My bad, thought I'd done that.

Comment (ericcornelissen) [reply icons/xiaomi.svg]: No problem, in fact my suggestion was [not correct either](https://travis-ci.com/simple-icons/simple-icons/jobs/242298724#L233) 😅 I fixed that for you in 3601733

**Comment by ericcornelissen** [comment]
Thanks a lot for your work on this @mudlabs 🎉 It is very much appreciated!

**Pull request merged**